build/package/scripts/build-typescript.sh

```diff
@@ -56,8 +70,24 @@ if [ -n "${NEXUS_HOST}" ] && [ -n "${NEXUS_USERNAME}" ] && [ -n "${NEXUS_PASSWOR
     npm config set strict-ssl=false
 fi;
 
-echo "Building ..."
+echo "Installing dependencies ..."
 npm ci
+
+echo "Linting ..."
+set +e
+npx eslint src --ext "${LINT_FILE_EXT}" --format compact --max-warnings "${MAX_LINT_WARNINGS}" > eslint-report.txt
```

Comment:
I guess if I use another file extension (e.g. `*.svelte`) in my project, I'd need to create and pass one string containing the default ones **plus** my specific one(s), right? Then I'd more prefer to have the default ones at place always but pass custom file extensions as an additional parameter (and only them).

Reply:
Could there be any reason why one would like to exclude some of the default ones? I would actually prefer to have this fully explicit, I also believe this is a bit easier to document and reason about. 

BTW: Would anything speak against just adding `*.svelte` to the default as well? It looks like there wouldn't be any harm, right?

Reply:
To me it makes sense to include `*.svelte` files as well and to also allow for users to not want to check certain extensions that are checked by default